internal/services/user/user.go

```diff
@@ -42,6 +42,7 @@ func NewService(ctx context.Context, userManager *users.Manager, brokerManager *
 func (s Service) GetUserByName(ctx context.Context, req *authd.GetUserByNameRequest) (*authd.User, error) {
 	name := req.GetName()
 	if name == "" {
+		log.Warningf(ctx, "GetUserByName: no user name provided")
```

Comment:
I find it difficult to find a good way to replace the function name in the warning/error message in this service. IMO we should somehow indicate the operation which this message is about. Just printing "no user name provided" is less helpful, because the context is completely unclear.

One option I see is to use `getpwnam: no user name provided` instead. `getpwnam` is the function which our NSS module hooks, which causes the `GetUserByName` method to be called. I'm unsure if it's better though. On the one hand, `getpwnam` is more cryptic to users than `GetUserByName`. On the other hand, if they search for `getpwnam`, they find some information about it, which might help them debug why it's called without a username on their system.

Reply:
@denisonbarbosa what do you think about this? should we leave the `GetUserByName:` prefix in these cases, use `getpwnam:` instead or do you see a better option?